server/service/vpp.go

```diff
@@ -128,7 +129,7 @@ func (r updateAppStoreAppResponse) Error() error { return r.Err }
 func updateAppStoreAppEndpoint(ctx context.Context, request interface{}, svc fleet.Service) (fleet.Errorer, error) {
 	req := request.(*updateAppStoreAppRequest)
 
-	updatedApp, err := svc.UpdateAppStoreApp(ctx, req.TitleID, req.TeamID, fleet.AppStoreAppUpdatePayload{
+	updatedApp, activity, err := svc.UpdateAppStoreApp(ctx, req.TitleID, req.TeamID, fleet.AppStoreAppUpdatePayload{
```

Comment:
Because we update the app itself and its auto-update schedule using separate service methods, we can't gather all of the info we need for the activity inside `UpdateAppStoreApp()` anymore. So I updated to that method to return an activity that we can then send after all of the updates are done.